components/build/jvm-build-service/system-config.yaml

```diff
@@ -8,11 +8,11 @@ data:
   image.cache: quay.io/redhat-appstudio/hacbs-jvm-cache:a4787c4c72c7f606515e19dc3057e79779b4f5a1
   builder-image.names: jdk11,jdk8,jdk17
   builder-image.jdk8.image: quay.io/redhat-appstudio/hacbs-jdk8-builder:a4787c4c72c7f606515e19dc3057e79779b4f5a1
-  builder-image.jdk8.tags: jdk:8,maven:3.8,gradle:7.5
   builder-image.jdk11.image: quay.io/redhat-appstudio/hacbs-jdk11-builder:a4787c4c72c7f606515e19dc3057e79779b4f5a1
-  builder-image.jdk11.tags: jdk:11,maven:3.8,gradle:7.5
   builder-image.jdk17.image: quay.io/redhat-appstudio/hacbs-jdk17-builder:a4787c4c72c7f606515e19dc3057e79779b4f5a1
-  builder-image.jdk17.tags: jdk:17,maven:3.8,gradle:7.5
+  builder-image.jdk8.tags: jdk:8,maven:3.8,gradle:7.4.2;6.9.2;6.9.2;5.6.4;4.10.2
```

Comment:
I may have missed the reason why in the super long slack thread last night @stuartwdouglas @dwalluck but along with adding a bunch of older gradle versions you removed 7.5

Just making sure that was intentional and not an oversight

Reply:
And latest version of 4 is 4.10.3.

Reply:
7.5 will not build ASM, 7.4 will. We are probably going to need even more versions.